task_DEV_5/task_DEV_5/FileGenerator.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Text;
+using System.IO;
+
+namespace task_DEV_5
+{
+    class FileGenerator
+    {
+        private const string INVALID_LINE_LENGTH_ERROR_MESSAGE = "String length cannot be negative.";       
+        private const int MAX_STRING_LENGTH = 101;
+        private const int MIN_STRING_LENGTH = 1;
+        private string path;
+        private int stringAmount;
+
+        public FileGenerator(string path, int stringAmount)
+        {
+            this.path = path;
+            this.stringAmount = stringAmount;
+        }
+
+        private void CheckStringLength()
```

Comment:
Private methods should be one by one after public methods.

Reply:
Done.